musicAlbum/genre.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './musicalbum'
+
+class Genre
+  attr_accessor :items, :name
+
+  def initialize(name)
+    @id = Random.rand(1...1000)
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items.push(item)
+    item.genre = self
+  end
```

Comment:
- [ ] Since you have the association for the item( genre, author, source, label) we should see all this data in the book and the movies and also in the music. please add the missing data in those files to make your app complete.

![Capture web_1-7-2022_122952_gitlab com](https://user-images.githubusercontent.com/61562626/176887302-42a4d273-9f63-43ff-bf09-a1fc21e59acf.jpeg)

![Capture web_1-7-2022_123835_gitlab com](https://user-images.githubusercontent.com/61562626/176887823-c358e560-e059-4594-bf17-89458feb2820.jpeg)